docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

```diff
@@ -330,6 +327,45 @@ On touch devices, TestCafe emulates touch events instead of mouse events.
 
 Mouse event | Touch event
 ----------- | -------------
-`mousemove` (when hovering or dragging) | `touchmove` (dragging only)
+`mousemove` (when hovering or dragging) | `touchmove` (when dragging)
 `mousedown` | `touchstart`
 `mouseup`   | `touchend`
+
+## Interaction Requirements
+
+TestCafe actions can interact with elements if they satisfy the following conditions:
+
+* an element is within the page's [`body`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/body) or in an [`<iframe>`](#work-with-iframes). The element can be invisible for the user. If the element is off-screen, TestCafe attempts to reach it with a scroll.
+
+* an element is visible - its properties are as follows:
+
+    Property | Value
+    -------- | --------
+    `display`  | *not* set to `none`
+    `visibility` | set to `visible` (the default value)
+    `width`    | auto calculated as or set to >= 1px
```

Comment:
What do you mean by "as or"?

Reply:
I meant to say

> **auto-calculated as >= 1px** or **set to >=px**

but wanted to cut the redundancies. I can see the flaw, please advise.

Reply:
`auto-calculated as >= 1px or set to >=px`

Thank you. Please apply your new suggestion.

Reply:
done